proof/refine/RISCV64/CNodeInv_R.thy

```diff
@@ -6944,6 +6944,7 @@ proof (induct rule: rec_del.induct,
                       red_zombie_will_fail.simps ball_simps(5))
   case (1 slot exposed)
   show ?case
+    supply opt_map_Some_comp[simp del] comp_apply[simp del]
```

Comment:
since you're in a `proof` block, you can use `note` to mark these as `[simp del]` for the entire lemma, which will save the repetition